### DISCUSSION TITLE
Unsuccessful test on sound

### DISCUSSION THREAD

**Discussion by hdhnl** [SpiralLinux/SpiralLinux-project#22] [Q&A]
In a fresh installation of spiralinux os, while the system and keyboard sounds and application sounds work, the sound test in the settings does not work.(gnome)
From https://linux-user.gr/t/anepitychhs-dokimh-ston-hcho-se-spiralinux-os-debian11-kai-alles-apories/4326

![image](https://user-images.githubusercontent.com/45640747/175135798-a61b8edf-07e2-4db9-990b-7d97457147b1.png)

**Comment by geckolinux** [comment]
Hi there, thanks for reporting. It would be interesting to test the vanilla Debian live image for Gnome to see if the same thing happens, just in case it's due to a missing package. But that's a common upstream issue that I've seen in many distros, not specific to SpiralLinux.

**Comment by hdhnl** [comment]
Hi @geckolinux, 
congratulations for your work.
On Debian vanilla live image for Gnome, the test works normally. 
From the Hellenic [LINUXUSER](https://linux-user.gr/) forum more experienced users than me, agree with you. In fact @dancaer69, thinks that as long as the sound works normally on SpiralLinux, as it does...
"Since the sound works everywhere, there is probably something wrong with the applet itself that does the sound test. Essentially it's playing back some audio files. So maybe those aren't installed or are in some other path."

**Reply by geckolinux** [reply]
Hi again @hdhnl , it turns out that you are correct. A user in this thread identified the missing package:
http://disq.us/p/2piqbnc
It will be included in the next set of ISOs.
Thanks for helping to make SpiralLinux better!

**Reply by hdhnl** [reply]
Hi @geckolinux,
I'm sincerely glad if I helped even a little.
But I in turn would like to thank you for the excellent SpiralLinux, 
[@Stelios_Haralabidis](https://linux-user.gr/u/Stelios_Haralabidis) for bringing SpiralLinux to the attention of the [Hellenic Linux forum,](https://linux-user.gr/) of which I am a [member](https://linux-user.gr/u/nikaskonstantinos) and [@dancaer69](https://linux-user.gr/u/dancaer69)
[@GNUTechie](https://linux-user.gr/u/GNUTechie) and
[@DarkGoth](https://linux-user.gr/u/DarkGoth)
for their answers and of course [Gary Rennie](https://disqus.com/by/disqus_PQKgR2Pisc/) that identified the missing package.

**Reply by geckolinux** [reply]
Excellent, thanks to all of you for your help.
